Makefile

```diff
@@ -17,6 +17,14 @@ HTML_FILES = $(shell find . -name \*.html)
 GO_FILES = $(shell find . -name \*.go)
 MD_FILES = $(shell find . -name \*.md)
 
+generate:
+	@go generate ./...
+.PHONY: generate
+
+generate-check: generate
```

Comment:
This is a CI check to ensure we don't forget to run `make generate` (or `go generate`) if metrics are changed.